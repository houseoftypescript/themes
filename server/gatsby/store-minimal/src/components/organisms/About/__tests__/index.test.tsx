import { render } from '@testing-library/react';
import React from 'react';
import { About } from '../index';

describe('About', () => {
  it('to match snapshot', () => {
    const { container } = render(<About />);
    expect(container).toMatchSnapshot();
  });
});
