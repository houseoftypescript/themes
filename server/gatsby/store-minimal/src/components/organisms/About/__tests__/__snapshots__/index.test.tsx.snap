// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About to match snapshot 1`] = `
<div>
  <section
    class="pb-8 md:pb-16"
    id="about"
  >
    <div
      class="MuiContainer-root MuiContainer-maxWidthLg css-1oqqzyl-MuiContainer-root"
    >
      <div
        class="flex flex-col gap-8"
      >
        <h1
          class="text-2xl"
        >
          About
        </h1>
        <p
          class="text-gray-700"
        >
          Lorem ipsum dolor sit amet, consectetur random link adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Vel risus commodo viverra maecenas accumsan lacus vel facilisis volutpat. Vitae aliquet nec ullamcorper sit. Nullam eget felis eget nunc lobortis mattis aliquam. In est ante in nibh mauris. Egestas congue quisque egestas diam in. Facilisi nullam vehicula ipsum a arcu. Nec nam aliquam sem et tortor consequat. Eget mi proin sed libero enim sed faucibus turpis in. Hac habitasse platea dictumst quisque. In aliquam sem fringilla ut. Gravida rutrum quisque non tellus orci ac auctor augue mauris. Accumsan lacus vel facilisis volutpat est velit egestas dui id. At tempor commodo ullamcorper a. Volutpat commodo sed egestas egestas fringilla. Vitae congue eu consequat ac.
        </p>
      </div>
    </div>
  </section>
</div>
`;
